diff --git a/auto/config b/auto/config
index c0d2f14..a6f6905 100755
--- a/auto/config
+++ b/auto/config
@@ -72,10 +72,10 @@ fi
 # Define options that vary across architectures
 case "$arch" in
     amd64)
-	lb_opts="$lb_opts --debian-installer cdrom"
+	lb_opts="$lb_opts --debian-installer live"
     ;;
     i386)
-	lb_opts="$lb_opts --debian-installer cdrom --linux-flavours 686-pae"
+	lb_opts="$lb_opts --debian-installer live --linux-flavours 686-pae"
     ;;
     armel|armhf)
 	lb_opts="$lb_opts --binary-images hdd --binary-filesystem ext4 --chroot-filesystem none"
